core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -229,6 +229,12 @@
     top: 14px;
   }
 
+  @media(max-width: 770px) {
+    .conversation-skin-tutor-card {
```

Comment:
Why?

Reply:
When the size of the browser is reduced, and the answer submitted is correct, the continue button (in the tutor card) hides under the Correct footer and the page cannot be scrolled down further.So this padding allows to scroll down the tutor card.

Reply:
OK. That's not obvious, so please add a comment in the code to explain. (Also, do this generally in the future if the "why" of something is not clear.)

Reply:
Sure!. Done!